kernel/src/syscall/open.rs

```diff
@@ -24,6 +24,10 @@ pub fn sys_openat(
         dirfd, path, flags, mode
     );
 
+    if path.is_empty() {
+        return_errno_with_message!(Errno::ENOENT, "openat fails with empty path");
```

Comment:
This is not documented in the man page. Could you provide a reference to support how this corner case is handled in this PR?